src/nanosvg.h

```diff
@@ -279,14 +279,17 @@ static void nsvg__parseElement(char* s,
 
 	// Get attribs
 	while (!end && *s && nattr < NSVG_XML_MAX_ATTRIBS-3) {
+		char* attr_ptr = NULL;
```

Comment:
let's call these `name` and `value`. That `_ptr` is not inline with the rest of the code naming style.